### DISCUSSION TITLE
how to get chat member list

### DISCUSSION THREAD

**Discussion by breadlol64** [eternnoir/pyTelegramBotAPI#1990] [General]
Please answer these questions before submitting your issue. Thanks!

1. What version of pyTelegramBotAPI are you using?
4.12.0
2. What OS are you using?
Windows 10
3. What version of python are you using?
3.11

I am need to get chat member list but i dont see method for it. Is there anytihing that can get chat member list?

**Comment by Badiboy** [comment]
Bot cannot get chat member list.

**Reply by unknownuserprofile** [reply]
But why, The owner can view member list
